src/theme.rs

```diff
@@ -558,7 +590,7 @@ pub trait Theme {
 }
 
 /// Default @clack/prompts theme.
-struct ClackTheme;
+pub struct ClackTheme;
```

Comment:
C'mon, no need to add `pub` at all
```suggestion
struct ClackTheme;
```

Reply:
Didn't even know I added it :see_no_evil: